rules/python_rules.build_defs

```diff
@@ -370,7 +370,7 @@ def pip_library(name:str, version:str, hashes:list=None, package_name:str=None,
         else:
             repo_flag = '-f ' + repo
 
-    target = pip_target_dir
+    target = f"{pip_target_dir}/{name}/{version}"
```

Comment:
yes, on the other hand you're right. If multiple versions are used that would have to reflect in the name.

Reply:
As for outs[0], the outs parameter is not used really anymore, just for that only change. So a proper way would be to remove the the outs and add something like subdirectory_name. Or make use of outs as a way to further filter down what the package outputs so that it does not take everything in. That could be filtered through jarcat's include/exclude options if needed.

Reply:
I've created #1202 to do this for  v16